pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,77 @@
+{ fetchFromGitHub
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
+, wrapGAppsHook
+, lmmath
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "plots";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "alexhuntley";
+    repo = "Plots";
+    rev = "v${version}";
+    hash = "sha256-GjNpaorxkkhZsqrKq4kO5nqF5+4I4tmSc023AZpY8Sw=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+  ];
```

Comment:
A blank line between nativeBuildInputs and the next "block" to align with the rest of the file?